dev/tests/integration/testsuite/Magento/Catalog/Model/ProductTest.php

```diff
@@ -723,4 +723,32 @@ public function productWithBackordersDataProvider(): array
             [1, 1, true],
         ];
     }
+
+    /**
+     * @magentoDataFixture Magento/Catalog/_files/products_different_store_values.php
+     * @magentoDbIsolation disabled
+     * @magentoAppIsolation enabled
+     * @magentoConfigFixture current_store catalog/frontend/flat_catalog_product 1
+     */
+    public function testExistsStoreValueFlagForMultipleProducts() {
+
+        $descriptionProudct = $this->productRepository->get('store_description');
+        $nameProudct = $this->productRepository->get('store_name');
+
+        /** @var \Magento\Store\Model\StoreManagerInterface $storeManager */
+        $storeManager = \Magento\TestFramework\Helper\Bootstrap::getObjectManager()->get(\Magento\Store\Model\StoreManagerInterface::class);
```

Comment:
Please `use Magento\TestFramework\Helper\Bootstrap;` and `use Magento\Store\Model\StoreManagerInterface;` and replace it by alias